acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -105,6 +131,7 @@ object StripePayment {
      * Obtains the Stripe 'customerId' directly from Stripe.
      */
     fun getStripeCustomerId(customerId: String) : String {
+
         // https://stripe.com/docs/api/java#create_card_token
         Stripe.apiKey = System.getenv("STRIPE_API_KEY")
```

Comment:
This thing doesn't test if the STRIPE_API_KEY is non-null. It's clearly an error if it is, and that error should be detected and signalled.